config/secrets.yml

```diff
@@ -13,6 +13,7 @@
 development:
   secret_key_base: 04c0aa396262a7530a9f0ba4c979376fea7f43761a3e980cb1397eae7152f33ec41c3b118ce9da26542e19f02bedc89e4f2c06a861f8bda96bb42e2f626c6fa0
   shubox_js_id: '4711fd96-9be2-46d5-8fb9-6eef6fd24a4b'
+  debug: <%= ENV['DEBUG'] %>
```

Comment:
I don't know if I would put this here as it doesn't feel quite "secret"-y.

Thoughts on putting this in the base application controller and exposing it as a helper? That feels like a more representative location as it can then be used in both the controller and view layer.